.chloggen/rpc_registry.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
I think this is probably breaking instead of deprecation?

Reply:
hm, when it is deprecation then? it looks like they should go in pair if we rename things?

Reply:
Yeah it can only be one of them, and given breaking has more weight, I'd pick that one to be more visible that things changed and action is needed.